src/azure-cli/azure/cli/command_modules/sqlvm/custom.py

```diff
@@ -270,7 +272,7 @@ def sqlvm_create(client, cmd, sql_virtual_machine_name, resource_group_name, sql
 def sqlvm_update(instance, sql_server_license_type=None, sql_image_sku=None, enable_auto_patching=None,
                  day_of_week=None, maintenance_window_starting_hour=None, maintenance_window_duration=None,
                  enable_auto_backup=None, enable_encryption=False, retention_period=None, storage_account_url=None,
-                 storage_access_key=None, backup_password=None, backup_system_dbs=False, backup_schedule_type=None,
+                 storage_access_key=None, backup_password=None, backup_system_dbs=False, backup_schedule_type=None, sql_management_mode=None,
```

Comment:
>sql_management_mode=None [](start = 116, length = 24)

this means null for update?

Reply:
No, this means for update customer has the option to only pass Full (meaning they want to update from LightWeight or NoAgent -> Full). We will not be changing the current value that we get from the get call. If customer provides no value, we pass the same value the entity returns.